sketch.go

```diff
@@ -102,7 +102,7 @@ func newCmRow(numCounters int64) cmRow {
 }
 
 func (r cmRow) get(n uint64) byte {
-	return byte(r[n/2]>>((n&1)*4)) & 0x0f
+	return r[n/2] >> ((n & 1) * 4) & 0x0f
```

Comment:
is this:
 - `(r[n/2] >> ((n & 1) * 4)) & 0x0f`
or:
 - `r[n/2] >> (((n & 1) * 4) & 0x0f)`

It seems an extra parenthesis can add clarity for that - it might just be me though.

Reply:
I would go with (r[n/2] >> ((n & 1) * 4)) & 0x0f